Add NumFOCUS to website and readme (#363)
holoviz/holoviz: doc/about.rst

  many other government agencies and corporations contributing to the maintenance
  and development of all the HoloViz libraries.

> - |NFbadge|: In early 2023, NumFOCUS became the fiscal sponsor for HoloViz. NumFOCUS is a 

Reviewer: ```suggestion
- |NFbadge|: NumFOCUS has been the fiscal sponsor for HoloViz since May 2023. NumFOCUS is a 
```
